.circleci/config.yml

```diff
@@ -81,6 +81,39 @@ common-steps:
         make $PKG_NAME
         ls ~/debbuild/packaging/*.deb
 
+  - &installgitlfs
+    run:
+      name: Install Git LFS.
+      command: |
+        export GIT_LFS_VERSION=2.7.2
+        export GIT_LFS_CHECKSUM=89f5aa2c29800bbb71f5d4550edd69c5f83e3ee9e30f770446436dd7f4ef1d4c
+        wget https://github.com/git-lfs/git-lfs/releases/download/v$GIT_LFS_VERSION/git-lfs-linux-amd64-v$GIT_LFS_VERSION.tar.gz
+        sha256sum git-lfs-linux-amd64-v$GIT_LFS_VERSION.tar.gz | grep $GIT_LFS_CHECKSUM
+        tar xzf git-lfs-linux-amd64-v$GIT_LFS_VERSION.tar.gz
+        sudo mv git-lfs /usr/local/bin/git-lfs
+        git lfs install
+
+  - &addsshkeys
+    add_ssh_keys:
+      fingerprints:
+        - "e5:b5:6e:d0:4e:ce:52:40:33:30:5e:6f:c5:73:38:20"
+
+  - &commitworkstationdebs
+    run:
+      name: Commit workstation debs for deployment to apt-test-qubes.freedom.press
+      command: |
+        git clone git@github.com:freedomofpress/securedrop-dev-packages-lfs.git
+        cd securedrop-dev-packages-lfs
+
+        git config user.email "securedrop@freedom.press"
```

Comment:
The email address will need to change if/when we implement commit signing. Fine for now!